src/runtime_src/core/tools/xbmgmt2/flash/xspi.cpp

```diff
@@ -784,8 +796,8 @@ bool XSPI_Flasher::isFlashReady() {
         }
         //TODO: Try resetting. Uncomment next line?
         //XSpi_WriteReg(XSP_SRR_OFFSET, XSP_SRR_RESET_MASK);
-        std::this_thread::sleep_for(std::chrono::nanoseconds(5000));
-        delay += 5000;
+        delay(std::chrono::microseconds(5));
```

Comment:
I like what you did here....it is clear to the user that we are delaying 5 microseconds.